tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -152,13 +152,21 @@ public void importGraphDef(byte[] graphDef, String prefix) throws IllegalArgumen
    * <p>
    * If {@code dx} is null, the implementation will use dx of {@link org.tensorflow.op.core.OnesLike OnesLike} for all
    * shapes in {@code y}.
+   * <p>
+   * {@code prefix} is used as the name prefix applied to all nodes added to the graph to compute gradients. It must
+   * be unique within the provided graph or the operation will fail. 
+   * <p>
+   * If {@code prefix} is null, then the nodes will be added to under the default prefix, which is "gradients" for the 
```

Comment:
I wouldn't promise what the implementation would do. How about just saying "If prefix is null, then one will be chosen automatically".